booleans.js

```diff
@@ -0,0 +1,124 @@
+function negate(a) {
+  return !a;
+};
+
+function both(a, b) {
+  if (a == true && b == true) {
+    return true;
+  } else {
+    return false;
+  };
+
+};
+
+function either(a, b) {
+  if (a == true | b == true) {
+
+    return true;
+  } else {
+
+    return false;
+  };
+};
+
+function none(a, b) {
+  return negate(either(a, b));
```

Comment:
See what you did here, very efficient! If these functions were not here though, you would need a way to do this.

As it happens you can use the same logic described above:
```
  return !(a || b);
```
As a bonus this will be more performant, as it involves fewer function calls. Points for thinking round corners though and reusing existing functions, I like that